doc/ssl/SSL_CTX_set_tlsext_status_cb.pod

```diff
@@ -16,8 +21,10 @@ Certificate Status Request functions
  long SSL_CTX_set_tlsext_status_arg(SSL_CTX *ctx, void *arg);
 
  long SSL_CTX_set_tlsext_status_type(SSL_CTX *ctx, int type);
+ long SSL_CTX_get_tlsext_status_type(SSL_CTX *ctx);
 
  long SSL_set_tlsext_status_type(SSL *s, int type);
+ long SSL_get_tlsext_status_type(SSL *s);
```

Comment:
@mattcaswell I would change the API:

```
/* Returns 1 on success, 0 on failure. */
int SSL_CTX_set_tlsext_status_type(SSL_CTX *ctx, int type);
/* Returns the type, or TLSEXT_STATUSTYPE_undef if no type is set. */
/* (TLSEXT_STATUSTYPE_undef defined to be -1 (or 0? the enum starts at 1 so will probably never have a 0 type) */
int SSL_CTX_get_tlsext_status_type(const SSL_CTX *ctx);
```

What is supremely odd usability-wise is to set an int but return a long for the same magic value. Changing the setter return from long to int is less crucial (most callers will check the value implicitly and will never see the type) though going `long` -> `int` should be safe, because code that uses long will continue to work. Hm, except maybe bindings... Anyway, the bottom line is that the implementation this API uses underneath shouldn't determine the API, and should be free to change in 1.1.0 or 1.2.0 or whenever. The caller of `SSL_get_tlsext_status_type` shouldn't have to care about SSL_ctrl. 

But of course since it's one of many SSL_ctrl victims, there needs to be a consistent plan for all of them...